en/db-migrations.md

```diff
@@ -9,5 +9,133 @@ keywords: 'database, migrations, schema, tables, columns'
 <hr/>
 ![](/assets/images/document-status-stable-success.svg) ![](/assets/images/version-{{ page.version }}.svg)
 
-> **NOTE**: Phalcon migrations have been removed from DevTools and moved to a separate repository. The package requires a lot of work and will be updated to the v4 version. As soon as the package is release ready we will update this document accordingly.
+> **NOTE**: Phalcon migrations have been removed from DevTools and moved to a separate repository.
 {: .alert .alert-info } 
+
+## Package git repository
+
+https://github.com/phalcon/migrations
+
+## Requirements
+
+* PHP >= 7.2
+* Phalcon >= 4.0.0
+
+## Installing via Composer
+
+```
+composer require --dev phalcon/migrations
+```
+
+```json
+{
+    "require": {
+        "phalcon/migrations": "^1.1"
+    }
+}
+```
```

Comment:
Could you explain what this means? User should create a file or modify an existent one? Right now this code isn't clear and IMO it is redundant: to getting help on Composer usage users should go to Composer documentation.

Reply:
e78f7ae